uxarray/grid/area.py

```diff
@@ -35,6 +41,9 @@ def calculate_face_area(
     coords_type : str, optional
```

Comment:
What happen to the input xyz if the `coords_type` is spherical?

Reply:
see below, as you point out.

Reply:
my point is it will be confusing if the coords_type is spherical but the input name is x y z (but in spherical you will only have lat and lon)

Either change the parameters name or at least put some detailed description in the docstring

Reply:
I get it, just that it'd be good for another PR. See usage [here](https://github.com/UXARRAY/uxarray/pull/1120/files#diff-b49076b32f4b332dcc526498a5e8d299bc1f870d843b9599b5a949804386fcabR61)

Reply:
Please at least include a detailed docstring specifying the limitations here. Otherwise it will create confusion in the future development.

Reply:
Done, added a TODO.